backends-velox/src/test/scala/io/glutenproject/execution/VeloxTPCHSuite.scala

```diff
@@ -271,3 +273,23 @@ class VeloxPartitionedTableTPCHSuite extends VeloxTPCHSuite {
     super.afterAll()
   }
 }
+
+class VeloxTPCHV1GlutenBhjVanillaBeSuite extends VeloxTPCHSuite {
+  override protected def sparkConf: SparkConf = {
+    super.sparkConf
+      .set("spark.sql.sources.useV1SourceList", "parquet")
+      .set("spark.sql.autoBroadcastJoinThreshold", "30M")
+      .set("spark.gluten.sql.columnar.broadcastJoin", "true")
+      .set("spark.gluten.sql.columnar.broadcastExchange", "false")
+  }
+}
+
+class VeloxTPCHV1VanillaBhjGlutenBeSuite extends VeloxTPCHSuite {
+  override protected def sparkConf: SparkConf = {
+    super.sparkConf
+      .set("spark.sql.sources.useV1SourceList", "parquet")
+      .set("spark.sql.autoBroadcastJoinThreshold", "30M")
+      .set("spark.gluten.sql.columnar.broadcastJoin", "false")
```

Comment:
Should there be one more test case where both the configs `broadcastJoin` and `broadcastExchange` are enabled ?

Reply:
The other existing suites should already cover the case you mentioned. I'll add some explicit config settings to those suits to make it clear.